bfe.json

```diff
@@ -0,0 +1,62 @@
+[
+  {
```

Comment:
There's nothing bad with this PR, it's very decent, but I think this is a suitable opportunity to change the format of this JSON. It doesn't make a lot of sense that it's an array, because then we use `json[1].code` to extract the "T" of messages, which... surprise surprise... is `1`.

I think a better format for this JSON would be objects like:

```json
{
  "feeds": {
    "code": 0,
    "formats": {
      "ssb/classic": {
        "code": 0,
        "etc": "etc",
      }
    }
  }
}
```

What do you think? Then we can use it like `json["feeds"].code` and `json["feeds"]["ssb/bendy-butt"].code`

Reply:
Yeah that would fix these magic constants. I have to check what changes are needed in tribes and envelope-js for this to work.

Reply:
I thought why not have both, in the other libraries both are useful depending on the context.